.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Description
+
+Please include a summary of the change and what type of change it is (new feature, bug fix, refactoring, documentation).
+Please also include relevant motivation and context.
+List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+### Quick checks:
+
+- [ ] There is no other [pull request](https://github.com/conduitio-labs/conduit-connector-activemqclassic/pulls) for the same update/change.
```

Comment:
WDYT about renaming the connector to conduit-connector-activemq-classic (with a dash before classic)? It helps a bit with readability.